roscala/src/main/scala/coop/rchain/roscala/ob/Template.scala

```diff
@@ -0,0 +1,19 @@
+package coop.rchain.roscala.ob
+
+import coop.rchain.roscala.ob.expr.TupleExpr
+
+class Template(val keyTuple: Tuple, val pat: CompoundPattern, val keyMeta: Meta) extends Ob {
+  def `match`(argvec: Tuple, nargs: Int): Option[Tuple] =
```

Comment:
Not sure if this a good idea to use `match` as a name of the method.

Reply:
Why not?